app/app.go

```diff
@@ -683,7 +716,8 @@ func New(
 		genutiltypes.ModuleName, evidencetypes.ModuleName, authz.ModuleName,
 		feegrant.ModuleName, nft.ModuleName, group.ModuleName,
 		paramstypes.ModuleName, upgradetypes.ModuleName, vestingtypes.ModuleName,
-		// icatypes.ModuleName,
+		ibchost.ModuleName,
```

Comment:
icbhost.ModuleName is already added above https://github.com/umee-network/umee/pull/1963/files#diff-0f1d2976054440336a576d47a44a37b80cdf6701dd9113012bce0e3c425819b7L680